src/main/java/com/google/devtools/build/lib/bazel/rules/java/jdk.WORKSPACE

```diff
@@ -171,3 +171,12 @@ http_archive(
     build_file = "@local_jdk//:BUILD.bazel",
     strip_prefix = "jdk10-server-release-1804",
 )
+
+http_archive(
+  name = "remote_java_tools",
+  sha256 = "321b437e0ad384b54c66f5acafcccdf87c7ae0fb85fa69e57d98f06042e931e4",
+  urls = [
+    "https://mirror.bazel.build/bazel_java_tools/java_tools_pkg.tar"
```

Comment:
I'm confused, aren't you creating a zip here? https://github.com/bazelbuild/bazel/pull/7155

If you want a tar, you should use tar.gz, but then you would have to split up by platform since in windows we typically use zips.

Also consider including a version information in the URL, otherwise when you change the package, it will break all builds of people who haven't synched yet.

Reply:
I am creating 2 archives:
* A zip archive that only contains the sources of the Java tools
* A tar archive that only contains the deploy jars created using the [`pkg_tar`](https://docs.bazel.build/versions/master/be/pkg.html#pkg_tar) rule (Also in #7155, defined in [third_party/BUILD](https://github.com/bazelbuild/bazel/pull/7155/files#diff-b6d6ae0d41e460758b3217d7fd82094c))

Why is tar.gz preferred over tar?

Reply:
I replaced this with a tar.gz (size goes down from 56M to 42M).

I checked with Laszlo and http_archive uses a Java implementation for decompressing the archive, that means we don't have to worry about compatibility. 

I also added a version in the URL.